examples/pipelines/controlnet-interior-design/components/generate_prompts/src/main.py

```diff
@@ -114,6 +114,17 @@ def load(self) -> dd.DataFrame:
 
         df = dd.from_pandas(pandas_df, npartitions=1)
 
+        # TODO remove, just use a tiny df for testing purposes
+        data = {
+            "prompts_text": [
+                "comfortable bathroom, art deco interior design",
+                "comfortable bathroom, bauhaus interior design",
+            ]
+        }
+        pandas_df = pd.DataFrame.from_dict(data)
+        df = dd.from_pandas(pandas_df, npartitions=1)
+        # end of TODO
```

Comment:
Would keep this as long as we're building the pipeline. Would remove once the pipeline is finished